src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,37 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService manufacturerService =
+            (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
     public static void main(String[] args) {
-        // test your code here
+        Driver driverPetr = new Driver("Petr","12345");
+        Driver driverBob = new Driver("Bob","23456");
+        System.out.println(driverService.create(driverBob));
+        System.out.println(driverService.create(driverPetr));
+        System.out.println(driverService.get(1L));
+        System.out.println(driverService.delete(2L));
```

Comment:
take ids from driverService.create(driverBob) don't use magic numbers as there is no guarantee that they are present in db.

Reply:
done